.github/workflows/bc.yml

```diff
@@ -0,0 +1,15 @@
+on:
```

Comment:
It will fail until there's a release. Let's remove it for now.

Reply:
Done.